### PR TITLE
Change German Umlauts mappings ("ä" -> "ae")

### PR DESCRIPTION
**Pull request by nutztherookie**
This makes total sense in a German context.
However I'm also hoping for a little insight if this proposal is not accepted as to which languages would handle this differently.

### CODE REVIEW

File: django/contrib/admin/static/admin/js/urlify.js [hunk 1]
```diff
@@ -3,15 +3,15 @@
...
-        'Õ': 'O', 'Ö': 'O', 'Ő': 'O', 'Ø': 'O', 'Ù': 'U', 'Ú': 'U', 'Û': 'U',
-        'Ü': 'U', 'Ű': 'U', 'Ý': 'Y', 'Þ': 'TH', 'Ÿ': 'Y', 'ß': 'ss', 'à': 'a',
-        'á': 'a', 'â': 'a', 'ã': 'a', 'ä': 'a', 'å': 'a', 'æ': 'ae', 'ç': 'c',
+        'Õ': 'O', 'Ö': 'Oe', 'Ő': 'O', 'Ø': 'O', 'Ù': 'U', 'Ú': 'U', 'Û': 'U',
+        'Ü': 'Ue', 'Ű': 'U', 'Ý': 'Y', 'Þ': 'TH', 'Ÿ': 'Y', 'ß': 'ss', 'à': 'a',
+        'á': 'a', 'â': 'a', 'ã': 'a', 'ä': 'ae', 'å': 'a', 'æ': 'ae', 'ç': 'c',
         'è': 'e', 'é': 'e', 'ê': 'e', 'ë': 'e', 'ì': 'i', 'í': 'i', 'î': 'i',
         'ï': 'i', 'ð': 'd', 'ñ': 'n', 'ò': 'o', 'ó': 'o', 'ô': 'o', 'õ': 'o',
-        'ö': 'o', 'ő': 'o', 'ø': 'o', 'ù': 'u', 'ú': 'u', 'û': 'u', 'ü': 'u',
+        'ö': 'oe', 'ő': 'o', 'ø': 'o', 'ù': 'u', 'ú': 'u', 'û': 'u', 'ü': 'ue',
```
Comment (charettes) [inline django/contrib/admin/static/admin/js/urlify.js]: I think this might be an issue with words such as `aigüe` in French.

**Comment by aaugustin** [comment]
If memory serves, this was discussed on django-developers a few years ago and there was no consensus towards making this change.

**Comment by nutztherookie** [comment]
I'm not that proficient in JavaScript but wouldn't it be possible to at least do something like 
include a .js file that has a `var LATIN_MAP = { ... }`
and then in `urlify.js`
```javascript
if (LATIN_MAP == null){
var LATIN_MAP = { ...}
}
```

**Comment by nutztherookie** [comment]
nevermind my latest idea. i realized i could simply put a slightly modified `urlify.js` in my project-own `static/`-folder as long as it is sourced first.

**Comment by timgraham** [comment]
I also think I remember a discussion about this, but I can't find it.

**Pull request closed without merging**